doc/contributing/adding-v8-fast-api.md

```diff
@@ -0,0 +1,120 @@
+# Adding V8 Fast API
+
+Node.js uses [V8](https://github.com/v8/v8) as the JavaScript engine.
+In order to provide fast paths for functions that are called quite often,
+V8 proposes the usage of `fast api calls` that does not use any of the V8
+internals, but uses internal C functions.
```

Comment:
Could we call into libuv using the V8 Fast API?

Reply:
We can call `libuv` from fast API, but that means no `isolate`, `context`, or `environment`. Side note: Enabling fast APIs was one of the reasons for adding `simdutf`.